src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py

```diff
@@ -955,6 +955,21 @@
           text: az vm assign-identity -g MyResourceGroup -n MyVm --role Reader
 """
 
+helps['vm run-command'] = """
+    type: group
+    short-summary: (PREVIEW) Manage run commands on a Virtual Machine
```

Comment:
Check with @sptramer on the syntax for the preview label. His recent PR established a standard.

Reply:
Based on the [diff](https://github.com/Azure/azure-cli/pull/4346/files#diff-097ad97ce6bcf2e69f56c3456ff451b8) in @sptramer's PR, I believe I am conforming.